sdk/go/core/ydk/types/types.go

```diff
@@ -103,14 +103,22 @@ type CommonEntityData struct {
 	BundleYangModelsLocation	string
 
 	// dynamic data (internals)
-	Parent 						Entity
+	Parent 					Entity
+}
+
+func (e *CommonEntityData) GetPath() string {
```

Comment:
Is this method necessary? For the purpose of extracting the segment path from an Entity struct, this method exists
https://github.com/ygorelik/ydk-gen/blob/715bed51b22bf3cb9e1e8d69aee23f9ee2d9544e/sdk/go/core/ydk/types/types.go#L316

Reply:
Deleted. Will check-in with next commit due to code overlapping with project in work.